packages/core/onClickOutside/component.ts

```diff
@@ -1,8 +1,10 @@
 import { h, ref, defineComponent } from 'vue-demi'
 import { onClickOutside } from '@vueuse/core'
+import { RenderableComponent } from '@vueuse/core/_types'
```

Comment:
```suggestion
import { RenderableComponent } from '../types'
```

Please pull the latest branch, it's renamed to `types`